extensions/@shopgate-user-privacy/frontend/locale/en-US.json

```diff
@@ -1,8 +1,9 @@
 {
   "user": {
     "delete_account": "Delete my account",
-    "delete_account_success": "We received your request to delete your account.",
+    "delete_account_success": "Löschanfrage erhalten. Sie wurden ausgeloggt und wir werden Ihre Daten löschen, soweit das rechtlich möglich ist. Dieser Prozess kann einige Tage in Anspruch nehmen.",
```

Comment:
I believe this sentence belongs into the de-DE.json file, right!?

Reply:
Right. This is fixed now, and all the other strings are available.